pyproject.toml

```diff
@@ -40,6 +40,7 @@ pytest="7.4.3"
 opustrainer = {git = "https://github.com/hplt-project/OpusTrainer.git", rev="9133e1525c7ee37f53ea14ee6a180152bf7ea192"}
 pytest-clarity = "^1.0.1"
 requests-mock = "^1.11.0"
+sh = "^2.0.6"
```

Comment:
question: I see you're using this in tests but not in the main script. I don't really have an objection, but I'm curious why?

Reply:
I tried to use this lib for the main scripts before and couldn't figure out how to run a command having its name in a variable (compression_cmd) from its code and docs. I dug a little more now and it seems there's a way: https://martinheinz.dev/blog/98. + I would need to add pip requirements and installation for this step. Subprocess works fine for this.

Ideally we should establish a reliable way to call other programs from Python. This use case arises across many scripts we migrate. Maybe we can just pre-install some lib that supports all the use cases we need (like sh) and does the exit status check, error handling, output etc. with one line. I had some issues with getting an output from OpusTrainer when it fails https://github.com/mozilla/firefox-translations-training/blob/30195248c4094186238cf6ac98149d538caf5f82/pipeline/data/dataset_importer.py#L64